core/src/main/java/com/datastax/oss/driver/api/core/config/DefaultDriverOptionUtil.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.datastax.oss.driver.api.core.config;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+
+public class DefaultDriverOptionUtil {
+  /**
+   * Fetch a boolean from the {@link DriverConfigProfile} if it exists, otherwise return provided
+   * default.
+   *
+   * @param configProfile to fetch from.
+   * @param option to fetch.
+   * @param defaultValue fallback value if the option doesn't exist.
+   * @return the specified boolean option, or the default if it's not defined.
+   */
+  public static boolean getConfigBooleanIfDefined(
+      DriverConfigProfile configProfile, DefaultDriverOption option, boolean defaultValue) {
```

Comment:
My suggestion was to add those directly to `DriverConfigProfile`, as default methods: `getBoolean(DriverOption option, boolean default)`, etc.